public/data/SamRB-dev.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "Sadim Rahman Badhan",
+    "location": "Hamm, NRW, Germany",
+    "bio": "Electronic Engineering student and tech enthusiast with aspiration to be an ML/AI practitioner with some experience in web development. Looking forward to make contributions to open source projects.",
+    "avatar": "https://github.com/SamRB-dev.png",
+    "portfolio": "https://github.com/SamRB-dev",
+    "skills": [
+        "Python",
+        "C++",
+        "Linux",
+        "web development",
+        "jupyter Notebook"
+    ], 
+    "socials": {
```

```suggestion
    "social": {
```
change `socials` to `social`